apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -79,7 +80,12 @@ class SectionActionDropdown extends Component {
    * Returns the URL to the correct section to be edited
    */
   editRedirectUrl = (sectionId, isPl) => {
-    let editSectionUrl = '/sections/' + sectionId + '/edit';
+    let editSectionUrl;
+    if (experiments.isEnabled('teacher-local-nav-v2')) {
```

Comment:
We should use [this function](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherNavigation/TeacherNavFlagUtils.ts#L9) to determine flag status